### ISSUE TITLE
NoSuchFieldError crash on machine placement

### ISSUE THREAD

**Issue by bigbantha** [TheTemportalist/ThaumicExpansion#6]
Crashed when placing any of the machines. Playing with CoFHCore 3.0.2-267 and Thermal Expansion 4.0.1-182. Strangely, this resulted in both client- and server-crash logs being generated, which are both below.

====CLIENT====
java.lang.NoSuchFieldError: numGroup
    at com.temportalist.thaumicexpansion.common.tile.TEThaumicAnalyzer.<init>(TEThaumicAnalyzer.java:88)
    at com.temportalist.thaumicexpansion.common.block.BlockThaumicAnalyzer.func_149915_a(BlockThaumicAnalyzer.java:121)
    at cofh.core.block.BlockCoFHBase.createTileEntity(BlockCoFHBase.java:63)
    at net.minecraft.world.chunk.Chunk.func_150806_e(Chunk.java:850)
    at net.minecraft.world.chunk.Chunk.func_150807_a(Chunk.java:660)
    at net.minecraft.world.World.func_147465_d(World.java:451)
    at net.minecraft.item.ItemBlock.placeBlockAt(ItemBlock.java:206)
    at net.minecraft.item.ItemBlock.func_77648_a(ItemBlock.java:103)
    at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:129)
    at net.minecraft.client.multiplayer.PlayerControllerMP.func_78760_a(PlayerControllerMP.java:353)
    at net.minecraft.client.Minecraft.func_147121_ag(Minecraft.java:1435)
    at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1941)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:961)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:887)
    at net.minecraft.client.main.Main.main(SourceFile:148)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:286)
    at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:376)
    at org.multimc.EntryPoint.listen(EntryPoint.java:165)
    at org.multimc.EntryPoint.main(EntryPoint.java:54)

====SERVER====
java.lang.NoSuchFieldError: numGroup
    at com.temportalist.thaumicexpansion.common.tile.TEThaumicAnalyzer.<init>(TEThaumicAnalyzer.java:88)
    at com.temportalist.thaumicexpansion.common.block.BlockThaumicAnalyzer.func_149915_a(BlockThaumicAnalyzer.java:121)
    at cofh.core.block.BlockCoFHBase.createTileEntity(BlockCoFHBase.java:63)
    at net.minecraft.world.chunk.Chunk.func_150806_e(Chunk.java:850)
    at net.minecraft.world.chunk.Chunk.func_150807_a(Chunk.java:660)
    at net.minecraft.world.World.func_147465_d(World.java:451)
    at net.minecraft.item.ItemBlock.placeBlockAt(ItemBlock.java:206)
    at net.minecraft.item.ItemBlock.func_77648_a(ItemBlock.java:103)
    at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:505)
    at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:128)
    at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:389)
    at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556)
    at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
    at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
    at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
    at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
    at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
    at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
    at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:186)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
    at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

**Comment by wooby6** [comment]
This mod is incompatible with the latest "COFH core 3.0.2-262".

To get it to work please downgrade "CoFH Core 3.0.1-254" & related mods until the mod dev updates the mod, you get this crash because the field "numGroup" has been changed in the latest version.

@TheTemportalist the incompatibility issue is located in TEThaumicAnalyzer.java

**Comment by TheTemportalist** [comment]
I am working on the next version. For some reason though I decided to move to SemVer for Origin, and that is taking time that I very hard to find at the moment. I am aware of this bug, and it should not exist soon. Unfortunately, my working in Origin is delaying the next release of TEC. My apologies.

**Comment by MapleSight** [comment]
Can you give us a status update on this bug fix? Sorry if I'm rushing you but this bug makes your work unusable. :+1:

**Comment by TheTemportalist** [comment]
Haha, your fine! Sorry for the long delay. I am slowing getting more free time to work on modding. Hopefully Ill have a fix out soon. The delay is because I completely re-wrote Thaumic Expansion, removing the dependency on CoFH things, and moved into my Origin system, which I also have been moving to work with Semantic Versioning. Currently I have a somewhat stable version of TeC, and I just need things tested basically. Ill try to see if there are any more issues between TeC and Origin today. In the mean time, if anyone wants to test, builds are found here: http://jenkins.dries007.net/view/Temportal/

**Comment by MapleSight** [comment]
It looks like a good idea to move away from CoFH dependency... Since they are always changing things. I'll look forward to your work. :D

**Comment by TheTemportalist** [comment]
Thanks. I have it somewhat stable after yesterday, Jenkins is just being stubborn.

**Comment by wooby6** [comment]
I can confirm current jenkins build for "ThaumicExpansion-1.7.10-3.0.0-beta.22.jar" (requires: "Origin-1.7.10-5.0.0.286.jar")  is working without error on "forge 10.13.3.1407-1.7.10" & on the latest COFH core, it's Un-craftable but working.

in-order to power the analyser you need to directly attach a Arcane Apparatus & then power the Arcane Apparatus through normal methods, currently has only one upgrade slot (top-right)

**Comment by bookerthegeek** [comment]
My question to this then is this:
1. We are running the latest version of The Dark Trilogy modpack.
2. Does this update require both server and client, or can we update just the server?

**Comment by wooby6** [comment]
both since hes completely reworte them. (jekins builds: http://jenkins.dries007.net/view/Temportal/)
 however as stated above the items (3 in total) is uncraftable, so you will have to add your own recipes via minetweaker3/ modtweaker2 these are alpha builds (testing builds), since they are currently working I can assume that the mod-dev will be releasing them in a more polished state in a few months.

**Comment by TheTemportalist** [comment]
Hi! I actually (surprisingly) LOVE to see chatter in the github issues! :D
I will definitely be adding recipes, but I am also waiting on a new model of the Arcane Apparatus. I released beta builds because I needed testing to be done :)
Full fledged releases should not be too long off. Once I am confident _all_ the bugs have been patched, I will release a full release version.

Is there still an issue here, or can I close this issue?

**Comment by wooby6** [comment]
The issue may not exist in the beta build however since the full release version has yet to be released this issue Is still required for future reference.  And may be closed once it has been released.

**Comment by TheTemportalist** [comment]
okay, thank you for your input.

**Comment by shadoxxhd** [comment]
i got the same issue with 3.0.1: http://pastebin.com/iCi5kYrf

**Comment by TheTemportalist** [comment]
I haven't experienced any issues on my server with the release and latest cofh mods. Please re open this issue or create a new issue if troubles continue to exist.
